### DISCUSSION TITLE
Serbian Language Leader

### DISCUSSION THREAD

**Discussion by cvladan** [OHF-Voice/intents#400] [Language Leaders]
I would like to officially apply for the position as Serbian language leader as I've already started contributing days ago and would like to continue doing so.

I'm native Serbian speaker and have experience with internationalisation and localisation of software.

**Comment by cvladan** [comment]
I also noticed that Croatian translation is using a lot of my pioneering work on Serbian language, and l that's ok as languages are really quite similar...

... and I've finally got some independent help and guidance for my contributions.

**Comment by synesthesiam** [comment]
Thanks @cvladan! I'm adding you as a language leader now :slightly_smiling_face:

**Comment by synesthesiam** [comment]
@maretodoric please coordinate with @cvladan. We're asking new contributors to submit at least two PRs before coming a language leader :+1:

**Comment by cvladan** [comment]
@maretodoric predlažem da zajedno završimo ovo!

Osim toga, ako imaš volje, značilo bi i da se angažuješ [oko lokalizacije Home Assistent-a](https://developers.home-assistant.io/docs/translations) jer smo i tu kao Srbija vrlo tanki do sada. Samo 7% je prevedeno, a koliko sam ja video poslednji submit je bio baš davno za to. Koristi se platforma [Lokalise](https://lokalise.com/) koja se dosta lako koristi.

**Comment by maretodoric** [comment]
Može. Što se tiče lokalizacije, već sam ubacivao neke prevode tu, postadoh neaktivan zbog privatnih obaveza i stigao mi mail da mi je istekao trial. Sad vidim da taj trial nema veze sa ovim i da mogu da nastavim da doprinosim prevođenju pa ću se baciti na to.
Inače, već sam forkovao intents pa sam počeo da istražujem trenutne primere na engleskom i dodavaću na srpskom, ćirilicu i latinicu.

**Reply by cvladan** [reply]
Super!

##### HA lokalizacija

Hvala na trudu. Ja još nisam unosio ništa oko prevoda HA. Inače, kad počnem, da li da idem prvo na desktop app ili na android? Ustvari, sad vidim da na Android aplikaciji ni nema srpskog jezika :(

##### Rhasspy Intents

Uneo sam ja već neke intents pa su momci za hrvatski prevod pokupili sa naših primera.
Pošto su oni nakon toga nastavili sa radom, planiram još malo da sačekam pa da nam onda oni vrate uslugu :)

Inače, ovde mislim da je bolje da se ne bavimo latinicom. Pošto je Voice u pitanju, mislim da neće ni morati da se radi u dva alafabeta. Ali i ako bude, lako ćemo na kraju uraditi preslovljavanje sa ćirilice.

**Reply by JabaMS** [reply]
Zdravo, ja bih se ukljucio u ovu pricu i pomogao oko lokalizacije. 
S tim da ja mislim da bi mozda ipak bilo dobro da paralelno radimo i latinicu. Jeste da je voice u pitanju, medjutim ova lokalizacija nece raditi ljudima (kao meni recimo) koji su podesili HA na sr-lat.

**Comment by PengBG** [comment]
Ја волим ћирилицу, али већ дуго времена од имплементације гласовне функције "wisper" и даље не даје команде на ћирилици већ само на латиници, или барем ја не знам то да подесим

**Comment by ddxic** [comment]
Zdravo drustvo

Ja bih se isto vrlo rado pridruzio u prevodu 😃

**Comment by ddxic** [comment]
Hi @synesthesiam 
I would like to apply as a Serbian language leader 😃 I submitted some PRs and would like to continue contributing.

**Reply by cvladan** [reply]
I'm ok with that.

**Comment by cvladan** [comment]
@synesthesiam Given that I'm short on time, it might be best to pass this on to someone with more capacity, such as @ddxic , and promote him to language leader for the Serbian language.

**Comment by synesthesiam** [comment]
Thanks @cvladan! Adding @ddxic as a language leader: https://github.com/home-assistant/intents/pull/2800

**Reply by ddxic** [reply]
Thank you all! Looking forward to a good cooperation 💪🏻

**Reply by stefancvetkovic** [reply]
Hi guys, I'm here to help for Serbian language. 
Please ping me if you need my assistance. Maybe we can catch up on discord on some private channel? 

Cheers

**Reply by ddxic** [reply]
@stefancvetkovic Postovanje Stefane, hvala na ponudi!
Treba sigurno pomoc. Na Lokalise (https://developers.home-assistant.io/docs/translations/) ima jos puno posla. A naravno i na Intents moze neki PR. Zahvalni smo za svaku pomoc!
Pozdrav

**Reply by ddxic** [reply]
@synesthesiam I was recently removed from the Home Assistant organization. I don't know why. And now I'm unable to merge PR's as a language leader, example #3407. Is there something I missed? I couldn't find anything. Thank you for your help.

**Comment by PengBG** [comment]
Коначно сам нашао мало времена и прошао кроз Home Assistant Frontend, очистио ћирилични део од латиничних превода, било их је баш много. 


Serbian - је ћирилични превод
Serbian (Latin) - је латинични

Молим вас да проверите изабрани језик пре почетка превођења, хвала

Мени је најлакше да изаберем вишејезички модалитет и ту селектујем Енглески, Босански, Хрватски и оба наша писма па је много лакше прводити

**Comment by PengBG** [comment]
Прошао кроз Home Assistant Frontend и средио првих 1100 кључева када се сортирају А-Z за оба писма, још само 6295 кључева :)
Такође сам урадио превод до 100% за Music Assistant пошто га доста користим, за оба писма, у следећој верзији би требало да се појави, за сада је у бета верзији убачено. Укупно 568 кључева.
Сад гледам интенте за оба писма, и видим неке словне грешкице али се не разумем много у Github па ћу ваљда скапирати како да отворим неки ПР

**Comment by PengBG** [comment]
Првих 4000 кључева за "Home Assistant Frontend" сређено за оба писма, требало би да је исти превод на оба, мада може да се поткраде нека грешкица. Још 3565 кључева до краја
